test/e2e/crud.go

```diff
@@ -42,23 +42,8 @@ var _ = Describe("Federated types", func() {
 
 	tl := framework.NewE2ELogger()
 
-	typeConfigs, err := common.FederatedTypeConfigs()
-	if err != nil {
-		tl.Fatalf("Error loading type configs: %v", err)
-	}
-
-	// Find the namespace typeconfig to be able to start its sync
-	// controller as needed.
-	var namespaceTypeConfig typeconfig.Interface
-	for _, typeConfig := range typeConfigs {
-		if typeConfig.GetTarget().Kind == util.NamespaceKind {
-			namespaceTypeConfig = typeConfig
-			break
-		}
-	}
-	if namespaceTypeConfig == nil {
-		tl.Fatalf("Unable to find namespace type config")
-	}
+	typeConfigs := common.TypeConfigsOrDie(tl)
+	namespaceTypeConfig := common.NamespaceTypeConfigOrDie(tl)
```

Comment:
It looks like this line is no longer needed since `validateCrud()` no longer sets up the namespace controller fixture directly.